### PR TITLE
test show me the money

### PR DESCRIPTION
**Pull request by JooYongLee**
* =ok
* test
  * yosi

### CODE REVIEW

**Review by yongyongnim — COMMENTED** [review]
you so look good

**Review by yongyongnim — CHANGES_REQUESTED** [review]
code is so good??

File: testFolder/sometest.py [hunk 1]
```diff
@@ -1,4 +1,6 @@
 import numpy as np
 
+def main():
+    print(np.random.randn(10))
```
Comment (yongyongnim) [inline testFolder/sometest.py:5]: function nameing??

**Review by yongyongnim — CHANGES_REQUESTED** [review]

File: testFolder/sometest.py (hunk 1 above)
Comment (JooYongLee) [reply testFolder/sometest.py:5]: OK TEST

**Pull request merged**